aten/src/ATen/mps/MPSAllocator.h

```diff
@@ -263,27 +247,44 @@ class MPSHeapAllocatorImpl
 
   // interface exposed to at::Allocator
   id<MTLBuffer> malloc(size_t size, uint32_t usage);
+  // frees a buffer and returns it into buffer pool
   void free(void* ptr);
+  // releases all the cached buffers and their associated heaps
   void emptyCache();
-  // interface exposed to internal MPS operations
+  // returns true if buffer was allocated from the shared pool
   bool isSharedBuffer(void* ptr);
+  // get the requested non-aligned size of an MTL buffer
```

Comment:
Maybe a better name is getRequestedUnalignedBufferSize instead of the need to have a comment ?

Reply:
A descriptive comment always helps with readability with public interfaces. 
I can change the name to getUnalignedBufferSize() to make it less verbose.

Reply:
Updated the PR.